src/components/ProjectsPage/Projects.js

```diff
@@ -243,19 +296,29 @@ const Projects = ({
       fetchProjectFunctions={fetchProjectFunctions}
       fetchProjectModels={fetchProjectModels}
       fetchProjectRunningJobs={fetchProjectRunningJobs}
+      filterByName={filterByName}
       filteredProjects={filteredProjects}
+      filterMatches={filterMatches}
       handleCreateProject={handleCreateProject}
+      handleSearchOnChange={handleSearchOnChange}
+      isDescendingOrder={isDescendingOrder}
       isEmptyValue={isEmptyValue}
       match={match}
       nuclioStore={nuclioStore}
       projectStore={projectStore}
+      projectsSortOptions={projectsSortOptions}
```

Comment:
you can declare options in View component and if it's static data you don't need useMemo